src/WebJobs.Script/Workers/Rpc/Configuration/LanguageWorkerOptionsSetup.cs

```diff
@@ -72,9 +80,62 @@ public void Configure(LanguageWorkerOptions options)
                 }
             }
 
-            var configFactory = new RpcWorkerConfigFactory(configuration, _logger, SystemRuntimeInformation.Instance, _environment, _metricsLogger, _workerProfileManager);
+            HashSet<string> workers = GetWorkersAvailableForResolutionViaHostingConfig();
+            bool dynamicWorkerResolutionEnabled = Utility.IsDynamicWorkerResolutionEnabled(_environment, workers);
```

Comment:
Related to my other comment, this would ready cleaner as 


```suggestion
            bool dynamicWorkerResolutionEnabled = _environment.IsDynamicWorkerResolutionEnabled(workers);
```